app/src/test/java/org/simple/clinic/contactpatient/ContactPatientUpdateTest.kt

```diff
@@ -88,12 +88,12 @@ class ContactPatientUpdateTest {
         .whenEvent(OverdueAppointmentLoaded(appointment))
         .then(assertThatNext(
             hasModel(defaultModel.overdueAppointmentLoaded(appointment)),
-            hasNoEffects()
+            hasEffects(LoadCallResultForAppointment(overdueAppointment.uuid))
```

Comment:
I think this is something we need to take a look at later. Whether to load them in a blocking manner one after the other or do them in a single call, both the appointment and call result. For now we can go ahead with this 👍🏾